pandora-server-directory/src/database/mongoDb.ts

```diff
@@ -35,18 +35,305 @@ type MongoDbInit = Readonly<{
 
 export const MONGODB_SERVER_VERSION: string = '6.0.5';
 
+interface DbAutomaticMigration {
+	readonly dryRun: boolean;
+	readonly log: Logger;
+	readonly startTime: number;
+	success: boolean;
+	totalCount: number;
+	changeCount: number;
+}
```

Comment:
There are some rather huge changes to the MongoDB driver, unrelated to the DM changes and not even a separate commit to review them separately. Frankly I don't feel like reviewing this and I feel that database is critical enough component that I don't want to approve it simply on the fact that tests still pass~

Reply:
sure I split that commit so it is more clear what code is just moved and what is new code